Linux Build because of missing reference assemblies

When checking out the project and issuing a `dotnet build` on a linux system I get:
```
/home/dmg/dotnet/sdk/3.0.100-preview8-013379/Microsoft.Common.CurrentVersion.targets(1175,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.6.1 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/home/dmg/projects/public/gui.cs.git/Terminal.Gui/Terminal.Gui.csproj]
    287 Warning(s)
    1 Error(s)
```

I have a simple solution for this, which is to use:
https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/

I will submit a PR in a few minutes to resolve this

--- a/Terminal.Gui/Terminal.Gui.csproj
+++ b/Terminal.Gui/Terminal.Gui.csproj
@@ -60,6 +60,10 @@
   <ItemGroup>
     <PackageReference Include="NStack.Core" Version="0.11.0" />
     <PackageReference Include="System.ValueTuple" Version="4.4.0" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
   <ItemGroup>
   </ItemGroup>

